src/main/features/claimant-response/form/models/states-paid/partPaymentReceived.ts

```diff
@@ -0,0 +1,30 @@
+import { IsDefined, IsIn } from 'class-validator'
+import { ValidationErrors } from 'forms/validation/validationErrors'
+import { YesNoOption } from 'models/yesNoOption'
+
+export class PartPaymentReceived {
```

Comment:
This model and `ClaimSettled` are doing exactly same. There are three other model classes used for the same model representation. i.e. `AcceptCourtOffer`, `AcceptPaymentMethod`, `SettleAdmitted` etc. Probably we can create one common class to represent this and reuse. 

@ellie-harrison  This is not to be implemented as part of this PR. But good to look at it as separate PR.